ts/components/ui/FocusAwareStatusBar.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import { StatusBar, StatusBarProps } from "react-native";
+import { withNavigationFocus } from "react-navigation";
+
+const FocusAwareStatusBar = withNavigationFocus<StatusBarProps>(
```

Comment:
```suggestion
const FocusAwareStatusBar = withNavigationFocus<StatusBarProps>(
```
could you leave a comment describing what this component does and when we should use it?

Reply:
Sure! Added with e714d2525da71dee2c4890a3c14dcfea866f24e1.